src/main/scala/io/iohk/ethereum/consensus/Protocol.scala

```diff
@@ -13,49 +13,38 @@ sealed trait Protocol {
   def name: String
 
   /** Returns `true` if this is the standard Ethereum PoW consensus protocol (`ethash`). */
-  final def isEthash: Boolean =
-    this match {
-      case Ethash ⇒ true
-      case _ ⇒ false
-    }
+  def isEthash: Boolean
```

Comment:
:(

Reply:
this is not used anywhere, is it? as far as I see ifEthash uses a match now

Reply:
[Gone](https://github.com/input-output-hk/mantis/pull/410/commits/ee65603bd983cc41373b32c4442cdf9b1bed4968)!